src/main/java/com/teamc/fms/service/TimeSlotService.java

```diff
@@ -0,0 +1,22 @@
+package com.teamc.fms.service;
+
+import com.teamc.fms.dto.TimeSlotDto;
+import com.teamc.fms.entity.TimeSlot;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface TimeSlotService {
+
+    TimeSlotDto updateTimeSlot(String id, TimeSlotDto timeSlot);
+
```

Comment:
As agreed, we should return an entity and then map it in the controller
We should receive a record instead of (String id, TimeSlotDto timeSlot), the record will contain all the required data to update the timeslot (UpdateTimeslotCommand)
mapping from UpdateTimeslotDto to UpdateTimeslotCommand is done in controller
![image](https://github.com/web-development-course-way/fms/assets/31379298/4b1c3c18-dc3f-4a0d-9651-525c7d440989)